notebooks/official/generative_ai/text_embedding.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                time.sleep(3)
</pre><p>Why is this sleep here? Is it to limit the number of requests sent to the service? It would be nice to include a comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Added</p>